_topics/en/asicboost.md

```diff
@@ -0,0 +1,72 @@
+---
+title: ASICBoost
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Overt ASICBoost
+  - Covert ASICBoost
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Mining
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: AsicBoost---A Speedup for Bitcoin Mining
+      link: https://arxiv.org/abs/1604.00575
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Overt ASICBoost support for S9 miners
+    url: /en/newsletters/2018/10/30/#overt-asicboost-support-for-s9-miners
+
+  - title: "Bitcoin Core #15471 removes warning falsely triggered by use of overt ASICBoost"
+    url: /en/newsletters/2019/03/05/#bitcoin-core-15471
+
+  - title: "Question about block templates versus actual blocks: ASICBoost explains one difference"
+    url: /en/newsletters/2021/04/28/#why-does-the-mined-block-differ-so-much-from-the-block-template
+
+## Optional.  Same format as "primary_sources" above
+# see_also:
+#   - title:
+#     link:
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **ASICBoost** is a technique for specially constructing a Bitcoin
+  block header in order to reduce by about 15% the number of operations
+  necessary to find a proof of work.
```

Comment:
"a proof of work" sounds slightly strange to me. How about "a block with valid proof of work"?